lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -492,6 +492,17 @@ function _dosomething_reportback_get_permalink_copy_vars($user) {
  * Alters the signup form that is attached to permalink pages.
  */
 function dosomething_reportback_form_alter(&$form, &$form_state, $form_id) {
+
+  // Make sure a default run is added to the filter
+  if ($form['#id'] == 'views-exposed-form-reportback-files-page-1') {
```

Comment:
`dosomething_reportback_form_views_exposed_form_reportback_files_page_1_alter` still not workin :angry: :drupal: